pom.xml

```diff
@@ -106,7 +106,7 @@
     <beam.version>2.1.0</beam.version>
```

Comment:
Should this be updated to `2.2.0-SNAPSHOT` ?

Reply:
Yes, you're right. I was confused about which version identifier to bump per the directions I was following.

Reply:
Actually, I don't believe that to be the case now.

I'm trying to replicate the effect of this comparable PR. 

https://github.com/GoogleCloudPlatform/DataflowJavaSDK/commit/f0ccc77f8f255e0b3cc5b2d570ce43e347d7dabe

The version dependency for the SDK should reflect the snapshot for the future release. The POM has been refactored since that change, so the SDK version is derived from project.version, which is properly updated by the release plugin. When I run 'mvn help:effective-pom', I see the updated version identifier matching my expectations.